docs/extensibility/xaml-designer-extensibility-migration.md

```diff
@@ -80,6 +104,16 @@ if (type != null && buttonType != type.IsSubclassOf(buttonType))
 }
```

Comment:
Would you mind fixing the C# sample too? I'm pretty sure this was meant to say this:
`if (type != null && type.IsSubclassOf(buttonType))` or shorter to match your change:
`if (type?.IsSubclassOf(buttonType) == true)`